N3.js/src/main/scala/org/w3/banana/n3js/io/N3jsTurtleParser.scala

```diff
@@ -0,0 +1,45 @@
+package org.w3.banana
+package n3js
```

Comment:
the full package is still `org.w3.banana.n3js.io` and it lives in the `org/w3/banana/n3js/io` directory. Am I missing something?